pydatastructs/graphs/algorithms.py

```diff
@@ -548,3 +550,61 @@ def strongly_connected_components(graph, algorithm):
         "isn't implemented for finding strongly connected components."
         %(algorithm, graph._impl))
     return getattr(algorithms, func)(graph)
+
+
+def random_walk(graph, start, path_length, operation, seed=44):
+    '''
+    Executes random walk from the starting node and given graph.
+
+    Parameters
+    ==========
+
+    graph: Graph
+        The graph on which random walk needs to be executed.
+    start: GraphNode or it's subclasses
+        The node/vertex from where random walk will begin.
+    path_length:
+        The maximum number of nodes/vertices traversed by random walk.
+
+    Returns
+    =======
+
+    path: list
+        Python list of nodes after execution of random walk.
+
```

Comment:
Doc strings not updated.

Reply:
ping @czgdp1807 
I updated them in the recent commit.